cdc/generation.cc

```diff
@@ -299,38 +299,6 @@ bool should_propose_first_generation(const gms::inet_address& me, const gms::gos
     });
 }
 
-// non-static for testing
-size_t limit_of_streams_in_topology_description() {
-    // Each stream takes 16B and we don't want to exceed 4MB so we can have
-    // at most 262144 streams but not less than 1 per vnode.
-    return 4 * 1024 * 1024 / 16;
```

Comment:
What about a mixed cluster? This may have to wait for a while.

Reply:
As the commit message explains:
> New nodes no longer insert generations into the old generations table
(which stored the entire generation in a single cell), they use the new
format instead. It is thus no longer necessary to limit the sizes of
created generations.

A node that contains this commit will not introduce CDC generations in the old format. For the new format it will refuse to introduce a new generation until the cluster is fully upgraded due to the feature flag check.